src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -137,6 +142,44 @@ public OutputCommitter createCommitter(TaskAttemptContext context) throws IOExce
     return new BigQueryOutputCommitter(context, delegateCommitter);
   }
 
+  /**
+   * This method is copied from
+   * {@link ForwardingBigQueryFileOutputFormat#checkOutputSpecs(JobContext)} to override
+   * {@link BigQueryFactory} with {@link BigQueryFactoryWithScopes}.
```

Comment:
nit: We should raise a ticket against BigQuery connector to fix the scopes

Reply:
Actually BigQuery Hadoop Connector support has been discontinued now: https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/930